web/cypress/integration/pluginTemplate/create-edit-delete-plugin-template.spec.js

```diff
@@ -74,6 +75,8 @@ context('Create Configure and Delete PluginTemplate', () => {
 
     cy.contains('Submit').click();
     cy.contains('Next').click();
+    cy.contains(selector.pluginCard, 'basic-auth').should('be.visible');
+    cy.contains(selector.pluginTitle, 'Authentication').should('be.visible');
```

Comment:
We'd better add assertions to determine that other category plugins don't exist

Reply:
not exist or not visible?